zsh/configs/post/path.zsh

```diff
@@ -1,8 +1,10 @@
 # ensure dotfiles bin directory is loaded first
 PATH="$HOME/.bin:/usr/local/sbin:$PATH"
 
-# load rbenv if available
-if command -v rbenv >/dev/null; then
+# load ASDF, falling back to rbenv if not available
+if [ -d "$HOME/.asdf" ]; then
+  . $HOME/.asdf/asdf.sh
```

Comment:
I would assume that `asdf.bash` only works with bash, not Zsh.

Interestingly, Zsh _can_ understand Bash completion by running this line:

```
autoload -U bashcompinit && bashcompinit
```